Dockerfile

```diff
@@ -0,0 +1,72 @@
+# Based on OpenAI's mujoco-py Dockerfile
+
+FROM nvidia/cuda:10.0-runtime-ubuntu18.04 AS base
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections \
+    && apt-get update -q \
+    && apt-get install -y --no-install-recommends \
+    build-essential \
+    curl \
+    ffmpeg \
+    git \
+    libgl1-mesa-dev \
+    libgl1-mesa-glx \
+    libglew-dev \
+    libosmesa6-dev \
+    net-tools \
+    parallel \
+    python3.7 \
+    python3.7-dev \
+    python3-pip \
+    rsync \
+    software-properties-common \
+    unzip \
+    vim \
+    virtualenv \
+    xpra \
+    xserver-xorg-dev \
+    ttf-mscorefonts-installer \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -o /usr/local/bin/patchelf https://s3-us-west-2.amazonaws.com/openai-sci-artifacts/manual-builds/patchelf_0.9_amd64.elf \
+    && chmod +x /usr/local/bin/patchelf
+
+ENV LANG C.UTF-8
+
+RUN    mkdir -p /root/.mujoco \
+    && curl -o mjpro150.zip https://www.roboti.us/download/mjpro150_linux.zip \
```

Comment:
I've been using mujoco200 to build the expert and reward models currently on S3. No preference between the two right now, though I'll have to regenerate everything later on, which currently takes a few days. Is there a reason to favor 150 over 200?



(I noticed that you changed from 200 to 150 in the commit history.)

Reply:
Gym currently only supports MuJoCo 1.5. They seem to have added this restriction recently: https://github.com/openai/gym/blob/master/setup.py

Reply:
Yikes, it looks like MuJoCo 2.0 is missing contact forces in the Ant and Humanoid environments. 
https://github.com/openai/gym/issues/1541

Reply:
Ouch, that's a pretty bad bug!